lineplot/lineplot.app.js

```diff
@@ -55,7 +55,12 @@
                                 uriWithFilters += "&recorded_time::leq::" + UriUtils.fixedEncodeURIComponent(end_x);
                             }
                             var uri = uriWithFilters + "/" + trace.x_col + "," + trace.y_col + "@sort(recorded_time)?limit=" + $rootScope.limit;
-                            server.http.get(uri).then(function(response) {
+                            if($rootScope.params["pcid"])
+                                contextHeaderParams['pcid']=$rootScope.params["pcid"]
+                            if($rootScope.params["ppid"])
+                                contextHeaderParams['ppid']=$rootScope.params["ppid"]
+                                
+                            server.http.get(uri, { headers: contextHeaderParams }).then(function(response) {
```

Comment:
Same case as plot, we want to call `ERMrest._certifyContextHeader()` before this get request.

Reply:
Made required changes